src/Features/Core/Portable/ChangeSignature/ChangeSignatureCodeAction.cs

```diff
@@ -26,19 +29,19 @@ public ChangeSignatureCodeAction(AbstractChangeSignatureService changeSignatureS
         public override object? GetOptions(CancellationToken cancellationToken)
             => AbstractChangeSignatureService.GetChangeSignatureOptions(_context);
 
-        protected override Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(object options, CancellationToken cancellationToken)
+        protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(object options, CancellationToken cancellationToken)
         {
             if (options is ChangeSignatureOptionsResult changeSignatureOptions && changeSignatureOptions != null)
```

Comment:
```suggestion
            if (options is ChangeSignatureOptionsResult changeSignatureOptions)
```

If the first test is true, it can't be null.